modules/nixos/installation/base.nix

```diff
@@ -1,15 +1,36 @@
 { pkgs, config, lib, inputs, system, ... }:
 {
-  boot.kernelPackages = config.boot.zfs.package.latestCompatibleLinuxPackages;
-  environment.sessionVariables.NIXPKGS_ALLOW_UNFREE = "1";
+  # Whitelist wheel users to do anything
+  # This is useful for things like pkexec
+  #
+  # WARNING: this is dangerous for systems
+  # outside the installation-cd and shouldn't
+  # be used anywhere else.
+  security.polkit.extraConfig = ''
+    polkit.addRule(function(action, subject) {
+      if (subject.isInGroup("wheel")) {
+        return polkit.Result.YES;
+      }
+    });
+  '';
+
+  boot.kernelPackages = lib.mkForce config.boot.zfs.package.latestCompatibleLinuxPackages;
+
   environment.systemPackages = with pkgs; [
+    git
     glibcLocales
+    glxinfo
+    gparted
+    nano
+    rsync
+    vim
   ];
+
   i18n.supportedLocales = [ "all" ];
+
   networking.hostName = "snowflakeos";
-  nix.extraOptions = ''
-    experimental-features = nix-command flakes
-  '';
+  networking.wireless.enable = false;
```

Comment:
Is this right? I would expect the iso to have support for wireless connections.

Reply:
Weirdly yes, since this is used to enable `wpa_supplicant`, however we want to use `NetworkManager` instead.